install/opensuse-installation-guide.md

```diff
@@ -0,0 +1,32 @@
+# Install Kata Containers on openSUSE Leap 42.3
+
+> **Notes:**
+>
+> - Kata Containers packages are available for [openSUSE\*](https://www.opensuse.org/)
+>   (currently `x86_64` only).
```

Comment:
For consistency with the other docs, please can you drop the "42.3" from the title (in this doc and the other one) and change this to:

> - Kata Containers packages are available for [openSUSE\*](https://www.opensuse.org/)
￼>   version 43.2 (currently `x86_64` only).